stable/owncloud/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: owncloud
-version: 7.0.1
+version: 7.0.2
```

Comment:
I think this change is more than a patch version. I think theoretically should be a major because you are changing the values.yaml

Reply:
You are right, we released a major for this changes in the past.